contracts/Manageable.sol

```diff
@@ -24,15 +30,16 @@ contract Manageable is ERC20Helper, GovManager{
     uint public totalOfRatios;
 
     modifier tokenReady(bool status) {
-         require(status ? totalUnlocks != 0 : totalUnlocks == 0, "Unlock Data status error");
+        require(status ? totalUnlocks != 0 : totalUnlocks == 0, "Unlock Data status error");
```

Comment:
we can try to use Xor (^)
require(status ^ totalUnlocks == 0, "Unlock Data status error");

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/User/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/User/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>
<!--table
	{mso-displayed-decimal-separator:"\.";
	mso-displayed-thousand-separator:"\,";}
@page
	{margin:.75in .7in .75in .7in;
	mso-header-margin:.3in;
	mso-footer-margin:.3in;}
tr
	{mso-height-source:auto;}
col
	{mso-width-source:auto;}
br
	{mso-data-placement:same-cell;}
td
	{padding-top:1px;
	padding-right:1px;
	padding-left:1px;
	mso-ignore:padding;
	color:black;
	font-size:11.0pt;
	font-weight:400;
	font-style:normal;
	text-decoration:none;
	font-family:Calibri, sans-serif;
	mso-font-charset:0;
	mso-number-format:General;
	text-align:general;
	vertical-align:bottom;
	border:none;
	mso-background-source:auto;
	mso-pattern:auto;
	mso-protection:locked visible;
	white-space:nowrap;
	mso-rotate:0;}
-->
</style>
</head>

<body link="#0563C1" vlink="#954F72">



Status | Unlock==0 | status ?... | XOR
-- | -- | -- | --
0 | 0 | 0 | 0
1 | 0 | 1 | 1
0 | 1 | 1 | 1
1 | 1 | 0 | 0



</body>

</html>

Reply:
This would have been an elegant solution but unfortunately, bitwise operators don't work with bool.
https://ethereum.stackexchange.com/questions/6829/why-cant-we-do-bitwise-operations-on-bools